src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/temperature/ClusterDimensionalSummary.java

```diff
@@ -98,12 +105,16 @@ public ClusterDimensionalSummary(TemperatureDimension profileForDimension) {
         }
     }
 
-    public void setMeanTemperature(TemperatureVector.NormalizedValue meanTemperature) {
+    public void setVectorValues(TemperatureVector.VectorValues meanTemperature) {
         this.meanTemperature = meanTemperature;
     }
 
-    public void setTotalUsage(double totalUsage) {
-        this.totalUsage = totalUsage;
+    public void setAvgMetricValueUsedOverNodes(double avgMetricValueUsedOverNodes) {
+        this.avgMetricValueUsedOverNodes = avgMetricValueUsedOverNodes;
+    }
+
+    public void setTotalMetricsValueUsed(double totalMetricValueUsedOverCluster) {
+        this.totalMetricValueUsedOverCluster = totalMetricValueUsedOverCluster;
```

Comment:
Average and total are over all shards in the cluster, right ? We should add that in the name to make it clear ?